pkg/kubelet/userns/userns_manager.go

```diff
@@ -130,16 +135,35 @@ func (m *UsernsManager) readMappingsFromFile(pod types.UID) ([]byte, error) {
 }
 
 func MakeUserNsManager(kl userNsPodsManager) (*UsernsManager, error) {
+	kubeletMappingID, kubeletMappingLen, err := kl.GetKubeletMappings()
+	if err != nil {
+		return nil, err
+	}
+
+	// If there are no explicit mappings configured, then reserve the first block for the host.
+	off := 1
+	len := math.MaxUint32 / userNsLength
+
+	if kubeletMappingLen > 0 {
```

Comment:
If we make `kl.GetKubeletMappings()` return the "default" mapping when no custom one is found? It seems simpler. Let's say it returns the range 65536 to MaxID or something like that.

And here we just treat it the same way, no matter if it was configured by the getsubids or none was found, for us here this just returns the range to use.

Reply:
good idea.  I've changed it